Automattic/wp-calypso
# Plans: Redirect to Plan Overview page after free trial checkout

This pull request fixes https://github.com/Automattic/wp-calypso/issues/2236 by redirecting users on the `Plan Overview` page rather than the `Thank You` page once checkout for a free trial is complete, with a success message:

![screenshot](https://cloud.githubusercontent.com/assets/594356/12201294/bf6f6ed8-b623-11e5-9fa9-d9087c298be7.png)

#### Testing instructions

1. Run `git checkout update/free-trial-checkout` and start your server
2. Get a new test account and blog
3. Select a plan by clicking the `Start Free Trial` button on the [`Plans` page](http://calypso.localhost:3000/plans)
4. Click the `Start 14 Day Free Trial` button on the `Secure Payment` page
5. Check that you are redirected on the `Plan Overview` page
6. Check that you see and can dismiss the success message

#### Reviews
